Makefile

```diff
@@ -42,6 +42,9 @@ test-unit:
 test-system: build
 	@ZTEST_BINDIR=$(CURDIR)/dist go test -v -tags=system ./tests -args PATH=$(shell pwd)/dist
 
+test: build
```

Comment:
`test` seems like prime real estate for this special target... how about `test-run` or something?
also i think the `-tags=system` and `PATH=...` are unnecessary (independently ran into this and put up #728)

Reply:
> `test` seems like prime real estate for this special target... how about `test-run` or something?
> also i think the `-tags=system` and `PATH=...` are unnecessary (independently ran into this and put up #728)

I just pushed a change inspired by your comment here... now you can just say 
```
make TEST=TestZTest/suite/zar/ls 
```
using Makefile if-logic.  And I took out system tag and PATH.  What do you think?

Reply:
looks good. but i merged #728 so you have a confict now 😬